Updated BLAST/Nomad Books/Creeper Spores/Illuminations Wiki Website

Xiribidus opened the pull request.

Xiribidus committed:
Update nomadbooks.md

Xiribidus committed:
Update nomadbooks.md

Xiribidus committed:
Update blast.md

Xiribidus committed:
Update blast.md

Xiribidus committed:
Merge pull request #1 from Xiribidus/BLAST-Wiki-Update

Update blast.md

Xiribidus committed:
Update blast.md

Xiribidus committed:
Update nomadbooks.md

Xiribidus committed:
Update creeper-spores.md

Pyrofab commented on wiki/creeper-spores.md:
```diff
@@ -4,31 +4,38 @@ slug: creeper-spores
+## Wiki
 
-## Creeperlings
+### Creeperling's Appearance
```
```suggestion
### Creeperlings' Appearance
```

Pyrofab commented on wiki/creeper-spores.md:
```diff
@@ -4,31 +4,38 @@ slug: creeper-spores
+This mod changes the usual Creeper explosion from blowing up your world to infesting it! Creeper explosions will cast the "Creeper Spores" effect on all entities caught in the explosion. When this effect runs out, the affected entity is hit or dies there is a chance for a Creeperling to spawn.
 
-## Gamerules
+![CreeperSporesEffect](https://user-images.githubusercontent.com/83953120/123487475-40347d00-d60e-11eb-9d65-1f42cb4f640e.png){: .wiki}
```
Could you maybe put that image on the website itself ?

Pyrofab commented on wiki/creeper-spores.md:
```diff
@@ -4,31 +4,38 @@ slug: creeper-spores
+In Creative Mode, clicking an adult Creeper or a Creeperling with a Creeper Spawn Egg will summon a Creeperling as with most other mobs.
 
-## Miscellaneous
+![Creeperlings](https://user-images.githubusercontent.com/83953120/123487499-4aef1200-d60e-11eb-92fd-44753171a502.png){: .wiki}
```
Same here

Pyrofab commented on wiki/creeper-spores.md:
```diff
@@ -4,31 +4,38 @@ slug: creeper-spores
-- Creepers take double damage from fire (dry leaves burn quite well I hear)
-- Charged creepers still destroy terrain unless mobGriefing is set to false (farming heads is a dangerous business)
-- You can spawn a creeperling by using a creeper spawn egg on another creeper or creeperling
+### Creeperling's Attributes
```
```suggestion
### Creeperlings' Attributes
```

Xiribidus committed:
Update wiki/creeper-spores.md

Co-authored-by: Pyrofab <redstoneinfire@gmail.com>

Xiribidus committed:
Update wiki/creeper-spores.md

Co-authored-by: Pyrofab <redstoneinfire@gmail.com>

Xiribidus committed:
Creeper-Spores Wiki

Xiribidus committed:
Update creeper-spores.md

Xiribidus committed:
Illuminations Wiki Website

I asked Rat if it was ready to go to the website but he didn't answer so here you go!

Xiribidus committed:
Spawn-Lanterns Wiki

Xiribidus committed:
Merge branch 'master' into master

Pyrofab commented on wiki/illuminations.md:
```diff
@@ -7,40 +7,173 @@ layout: wiki
+This mod is compatible with the [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer).
 
-## Illuminations types
+*(All screenshots with a bloom effect were taken with Canvas Renderer)*
```
```suggestion
*(All screenshots with a bloom effect were taken with [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer) and [Lumi Lights](https://spiralhalo.github.io/)*
```

Pyrofab committed:
Add canvas and lumi lights links

Pyrofab reviewed (approved):
Excellent work, thank you very much !

Pyrofab commented on wiki/illuminations.md:
```diff
@@ -7,40 +7,173 @@ layout: wiki
+This mod is compatible with the [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer).
 
-## Illuminations types
+*(All screenshots with a bloom effect were taken with [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer) and [Lumi Lights](https://spiralhalo.github.io/)*
```
```suggestion
*(All screenshots with a bloom effect were taken with [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer))*
```

Pyrofab committed:
Actually no lumi lights

also forgot a parens

Pyrofab commented on wiki/illuminations.md:
```diff
@@ -7,40 +7,173 @@ layout: wiki
+This mod is compatible with the [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer).
 
-## Illuminations types
+*(All screenshots with a bloom effect were taken with [Canvas Renderer](https://www.curseforge.com/minecraft/mc-mods/canvas-renderer))*
```
```suggestion
*(All screenshots with a bloom effect were taken with Canvas installed)*
```
ok I'm stupid you linked it right above

Pyrofab committed:
Forget about everything
